### ISSUE TITLE
Cache clear, cannot declare class

### ISSUE THREAD

**Issue by KDederichs** [schmittjoh/JMSDiExtraBundle#303]
Hey,
I've been seeing a strange issue where the clear cache command cannot declare classes on it's first execution if there is no previous cache.

On the first execution it will always fail with something like 
`
PHP Fatal error:  Cannot declare class EnhancedProxy_84f26540d03882f66a8da2bedf47ccf4f27752f4\__CG__\TaskBundle\Controller\SupervisionController, because the name is already in use in /home/kai/public_html/washeroo/var/cache/tes_/jms_diextra/lookup_method_classes/TaskBundle-Controller-SupervisionController.php on line 11
14:33:41 CRITICAL  [php] Fatal Compile Error: Cannot declare class EnhancedProxy_84f26540d03882f66a8da2bedf47ccf4f27752f4\__CG__\TaskBundle\Controller\SupervisionController, because the name is already in use ["exception" => Symfony\Component\Debug\Exception\FatalErrorException { …}] []
`
On any subsequent run of the clean cache command it will be successful though.

I already posted it to symfony thinking it might be cache clear related but they said it"s not their bug.

As reference here is the issue:
https://github.com/symfony/symfony/issues/25576

Here is also something to reproduce said behavior:

https://github.com/KDederichs/symfony-standard/tree/issue_25576

**Comment by greg0ire** [comment]
Is this a duplicate of #96 ?

**Comment by KDederichs** [comment]
I don't think so as the error shown in #96 is a different one and this is not ACL related.
